Gems/ROS2/Code/Source/Frame/NamespaceConfiguration.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ROS2/Frame/NamespaceConfiguration.h"
+#include "ROS2/Utilities/ROS2Names.h"
+#include <AzCore/Serialization/EditContext.h>
+#include <AzCore/Serialization/EditContextConstants.inl>
+
+namespace ROS2
+{
+    void NamespaceConfiguration::PopulateNamespace(bool isRoot, AZStd::string entityName)
+    {
+        m_isRoot = isRoot;
+        m_entityName = AZStd::move(entityName);
```

Comment:
Member initializer list is preferred for constructor initialization.
```suggestion
    void NamespaceConfiguration::PopulateNamespace(bool isRoot, AZStd::string entityName)
        : m_isRoot(isRoot)
        , m_entityName(AZStd::move(entityName))
    {
```

Reply:
It is not a constructor.

Reply:
@lumberyard-employee-dm is there anything else to do with this item?

Reply:
Ah sorry about that.
I thought this was a constructor.